src/com/sun/ts/tests/compat12/ejb/Tx_Single/TestBeanEJB.java

```diff
@@ -26,7 +26,7 @@
 import java.util.*;
 import java.rmi.*;
 import jakarta.ejb.*;
-import javax.transaction.*;
+import jakarta.transaction.*;
```

Comment:
here is another conflict, this should be done already.

Reply:
There is not a conflict :) Check currnt master: https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/tests/compat12/ejb/Tx_Single/TestBeanEJB.java

line 29: import javax.transaction.*;